proto/src/determined/rbac/v1/rbac.proto

```diff
@@ -76,6 +76,14 @@ enum PermissionType {
   // Ability to delete experiment.
   PERMISSION_TYPE_DELETE_EXPERIMENT = 2006;
 
+  /* TODO: update the comments. we delegate tensorboard to view experiment artifacts. */
+  // Ability to create nscs.
+  PERMISSION_TYPE_CREATE_NSC = 3001;
+  // Ability to view nsc's.
+  PERMISSION_TYPE_VIEW_NSC = 3002;
+  // Ability to update nsc and nsc's lifecycle.
+  PERMISSION_TYPE_UPDATE_NSC = 3003;
```

Comment:
not sure if we need to separate out update from create.
anyone with create access should be able to delete ntsc

Reply:
also `NSC` is vague we'd want to clarify that if we keep these together

Reply:
that makes sense, though I can imagine someone being given delete permissions in order to clean things up - like an automated account that runs cleanup scripts monthly on old NSCs

Reply:
makes sense. that could be part of the update? I'm not sure we need an update permission for ntsc. there is nothing to update other than to kill.  By delete do you mean kill?